etc/conf/mtTelemetry.yaml

```diff
@@ -58,17 +63,40 @@ end_collection_event:
     device: MTCamera
     topic: endOfImageTelemetry
 
-## The event for the image readout paramters
-#imageParam_event:
-#    device: MTCamera
-#    topic: imageReadoutParameters
-#    array_keys: ccdLocation
+# The event for the camera vendor and CCD names configuration
+cameraConf_event:
+    device: MTCamera
+    topic: focalPlaneSummaryInfo
+    value: ccdManufacturer
+    array_keys: ccdLocation
 
 # Telemetry values for images meta-data
-# https://confluence.lsstcorp.org/display/SYSENG/MT+Header+Information+Topic+Mapping
+# https://confluence.lsstcorp.org/display/SYSENG/ComCam+Header+Information+Topic+Mapping
```

Comment:
URL looks to be wrong here. This should be MT. BTW, the `confluence.lsstcorp.org` URLs should be replaced with `rubinobs.atlassian.net` based URLs. Simply replacing the hostname doesn't work.

Reply:
Fixed